ulog/log.go

```diff
@@ -99,12 +107,57 @@ func TestingLogger() (Log, *spy.Sink) {
 	}, sink
 }
 
-// Logger returns the package-level logger configured for the service
-// TODO:(at) Remove Logger() call, _std and defaultLogger() access in ulog
-func Logger() Log {
+func logger() Log {
+	_setupMu.RLock()
+	defer _setupMu.RUnlock()
+
 	return &baseLogger{
-		log: _std.(*baseLogger).log,
+		log: _std.log,
+	}
+}
+
+// SetLogger sets configured logger at the start of the service
+func SetLogger(log Log) {
+	_setupMu.Lock()
+	defer _setupMu.Unlock()
+	_std = log.(*baseLogger)
```

Comment:
Its ok to panic here since SetLogger() is meant to be used in service setup only.